### PR TITLE
add anomaly heatmap to dataset item as metadata

### PR DESCRIPTION
**Pull request by djdameln**
Computes the anomaly heatmaps in the inference callback and adds it to the dataset item as a metadata.

### CODE REVIEW

File: external/anomaly/ote/callbacks/inference.py [hunk 1]
```diff
@@ -56,3 +60,9 @@ def on_predict_epoch_end(self, _trainer: pl.Trainer, _pl_module: AnomalyModule,
             )
 
             dataset_item.append_annotations([shape])
+
+            heatmap = anomaly_map_to_color_map(anomaly_map.squeeze())
+            heatmap_media = ResultMediaEntity(
+                name="Anomaly map", type="anomaly_map", annotation_scene=dataset_item.annotation_scene, numpy=heatmap
```
Comment (samet-akcay) [inline external/anomaly/ote/callbacks/inference.py:66]: maybe we could have the name as `name="Anomaly Map"` ?

**Review by samet-akcay — COMMENTED** [review]
Only a minor comment. Looks good otherwise.

**Comment by ashwinvaidya17** [comment]
Looks good to me. Thanks!

**Review by ashwinvaidya17 — APPROVED** [review]

**Review by samet-akcay — APPROVED** [review]
Thanks @djdameln, looks good!

**Pull request merged**